src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs

```diff
@@ -11037,5 +11039,76 @@ public void InvalidCharacterInPdbPath()
                     Diagnostic(ErrorCode.FTL_InvalidInputFileName).WithArguments("test\\?.pdb").WithLocation(1, 1));
             }
         }
+
+        [ConditionalFact(typeof(WindowsDesktopOnly), Reason = ConditionalSkipReason.NativePdbRequiresDesktop)]
```

Comment:
>[ConditionalFact(typeof(WindowsDesktopOnly), Reason = ConditionalSkipReason.NativePdbRequiresDesktop)] [](start = 8, length = 102)

Is there a reason to limit the test to Win desktop? This should work on all platforms.

Reply:
@tmat , neighbor tests apply these restrictions. And in the previous iteration, those tests failed in Linux and Mac only: https://dev.azure.com/dnceng/public/_build/results?buildId=383581&view=ms.vss-test-web.build-test-results-tab

Reply:
The C# ones did not fail. The VB ones failed because VB PDB tests don't normalize line endings yet (`WithWindowsLineBreaks`).